### PR TITLE
Add lazy.nvim installation instructions to README

### PR DESCRIPTION
**Pull request by Drahlous**
Just adding lazy.nvim instructions to the README to make it easier for folks to copy/paste into their config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +48,28 @@ use {
...
+
+```lua
+-- init.lua:
+{
+    "ThePrimeagen/harpoon",
+    branch = "harpoon2",
+    dependencies = { "nvim-lua/plenary.nvim" },
+    opts = {},
+    config = function()
+        local harpoon = require("harpoon")
```
Comment (daltondiaz) [inline README.md:61]: It's missing `harpoon:setup()`

I don't if Prime agree with this, but I think it's better to add harpoon configuration in folder `~/.config/nvim/lua/plugins/`, but maybe this is optional, more important is harpoon:setup

**Review by daltondiaz — CHANGES_REQUESTED** [review]
Missing harpoon:setup()

File: README.md (hunk 1 above)
Comment (Drahlous) [reply README.md:61]: Thank you for taking the time to review my PR!
I've added the `harpoon:setup()` call as suggested. I also agree that the config belongs in its own file in `~/.config/nvim/lua/plugins/`, so I've updated the filepath and added a return statement to reflect this.

**Comment by Drahlous** [comment]
Added harpoon:setup()

**Review by daltondiaz — APPROVED** [review]
It's fine for me, waiting other reviewer to approve

**Comment by ThePrimeagen** [comment]
i don't think i want more documentation on how to install yet another package manager.  right now, the important parts are not documented, but how to get started is a bunch of times.

**Comment by lpanebr** [comment]
Adding the proper way to setup harpoon2 on LazyVim even though this is closed. Just in case anybody stumbles here.

Create the file `plugins/harpoon.lua`:

```lua
return {
  "ThePrimeagen/harpoon",
  lazy = false,
  branch = "harpoon2",
  init = function()
    local harpoon = require("harpoon")

    -- REQUIRED
    harpoon:setup()
    -- REQUIRED

    vim.keymap.set("n", "<leader>a", function() harpoon:list():append() end)
    vim.keymap.set("n", "<C-e>", function() harpoon.ui:toggle_quick_menu(harpoon:list()) end)

    vim.keymap.set("n", "<C-h>", function() harpoon:list():select(1) end)
    vim.keymap.set("n", "<C-t>", function() harpoon:list():select(2) end)
    vim.keymap.set("n", "<C-n>", function() harpoon:list():select(3) end)
    vim.keymap.set("n", "<C-s>", function() harpoon:list():select(4) end)

    -- Toggle previous & next buffers stored within Harpoon list
    vim.keymap.set("n", "<C-S-P>", function() harpoon:list():prev() end)
    vim.keymap.set("n", "<C-S-N>", function() harpoon:list():next() end)
  end,
  dependencies = { "nvim-lua/plenary.nvim" },
}
```

**Pull request closed without merging**